simple_use_case/kadistudio/README.md

```diff
@@ -0,0 +1,90 @@
+# KadiStudio
+Implementation of the simple use case with KadiStudio. `KadiStudio` is a software component of the
+Kadi4Mat research data infrastructure, that is dedicated to model, visualise and execute scientific workflows using a GUI.
+
+## Installation
+`KadiStudio` can be installed using the debian package provided [here](https://kadi4mat.iam-cms.kit.edu/records/6479).
```

Comment:
I can't access that?

Reply:
Hi @joergfunger ,
I now provided a link to a bwsync&share repository that is freely accessible.
Please let me know if that works for you now.